### ISSUE TITLE
Allow hiding “Log in or create an account” notification

### ISSUE THREAD

**Issue by textpain** [getinsomnia/insomnia#6601]
### Expected Behavior

Allow hiding that annoying bar on top.
<img width="1318" alt="Screenshot 2023-09-29 at 10 56 10" src="https://github.com/Kong/insomnia/assets/14220138/3c824946-5089-4bd4-b95b-575b5df6a455">

### Actual Behavior

You don't seem to allow hiding it.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

Latest

### What operating system are you using?

macOS

### Operating System Version

Latest

### Installation method

insomnia.rest

**Comment by Windowsfreak** [comment]
+1 pretty annoying if creating an account or syncing to cloud is not an option
but I suspect @subnetmarco will just ignore all our anger and close all tickets

**Comment by textpain** [comment]
@Windowsfreak the thing is, you can use the new Insomnia just fine without syncing. It's just that this annoying thing discourages you from doing that. Back in the day, it was exactly the reason I switched from Postman to Insomnia — Postman did the exact same thing.

**Comment by subnetmarco** [comment]
We will introduce a way to remove that banner notification.

**Comment by textpain** [comment]
@subnetmarco nice to know! Can you give further info as in when do you plan this to happen? I'm actively researching Insomnia alternatives, just like other people who were disappointed with the updates, and the timeline will probably influence my decision.

**Comment by subnetmarco** [comment]
As soon as the next week, I already instructed the team to work on a dismissible "x" that will hide it.

We may have to fix another couple issues so we ship a new release with more density.

It will land in a 8.2 version.

**Comment by DusanLesan** [comment]
I was shocked after starting the new version and thought that I wouldn't be able to see my requests. However, after going through some unintuitive process shown in a video from GitHub issue comments, I managed to recover most of my data in a 'local scratchpad.' Having to create an account for my client's requests would be a deal breaker for me. No important data will be leaving my computer and my own backup storage.

That said, I did find a solution for the rapidly deteriorating UX of this app. Kong was kind enough to provide themes and access to the app's CSS. So, if you add this to the 'rawCss':
```
.app > div > div {
    grid-template: "Header Header" 0px
	"Banner Banner" 0px
	"Navbar Content" 1fr
	"Statusbar Statusbar" 1px [row-end] / 0px 1fr;
}
```
You will remove unnecessary bloat reducing the usable elements of the app and end up with this:
![inso](https://github.com/Kong/insomnia/assets/20520590/a19a0343-9446-4b20-bc3c-0e178828c4a3)
